lbrynet/extras/wallet/claim_proofs.py

```diff
@@ -32,7 +31,7 @@ def verify_proof(proof, rootHash, name):
                 if previous_child_character >= child['character']:
                     raise InvalidProofError("children not in increasing order")
             previous_child_character = child['character']
-            to_hash += six.int2byte(child['character'])
+            to_hash += bytes((child['character'], ))
```

Comment:
Extraneous space after the comma?

Reply:
The comma is needed so that `child['character']` gets interpreted as a `tuple` rather than an `int`:
```python
>>> u = ((3))
>>> u
3
>>> v = ((3, ))
>>> v
(3,)
>>> bytes(u)
b'\x00\x00\x00'
>>> bytes(v)
b'\x03'
```
The space preceding the comma makes it look less cluttered.

Reply:
Can you please remove the space after the comma.